testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java

```diff
@@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
   }
 
   @Override
-  public String getVMArguments(ILaunchConfiguration configuration)
-      throws CoreException {
-    StringBuilder vmArgs = new StringBuilder();
-    vmArgs.append(super.getVMArguments(configuration));
-
-    vmArgs.append(" ");
-    vmArgs.append(ConfigurationHelper.getJvmArgs(configuration));

```

Comment:
Just pushed a fix, should be version 51, can you confirm?

-- 
Cédric


On Wed, Dec 23, 2015 at 3:16 PM, Nick Tan <notifications@github.com> wrote:

> In
> testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java
> <https://github.com/cbeust/testng-eclipse/pull/212#discussion_r48386416>:
>
> > @@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
> >    }
> >
> >    @Override
> > -  public String getVMArguments(ILaunchConfiguration configuration)
> > -      throws CoreException {
> > -    StringBuilder vmArgs = new StringBuilder();
> > -    vmArgs.append(super.getVMArguments(configuration));
> > -
> > -    vmArgs.append(" ");
> > -    vmArgs.append(ConfigurationHelper.getJvmArgs(configuration));
>
> hi @cbeust <https://github.com/cbeust>
> looks like the testng.jar in the new plugin was built with java 8:
> java.lang.UnsupportedClassVersionError: org/testng/remote/RemoteTestNG :
> Unsupported major.minor version 52.0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cbeust/testng-eclipse/pull/212/files#r48386416>.
>